app/build.gradle

```diff
@@ -18,7 +18,7 @@ repositories {
 android {
     compileSdkVersion 24
     //noinspection GradleDependency
-    buildToolsVersion "23.0.2"
+    buildToolsVersion '25.0.0'
```

Comment:
I changed buildToolsVersion to 23 again but now gradlew is throwing an error 
`> The SDK Build Tools revision (23.0.2) is too low for project ':app'. Minimum required is 25.0.0
`